db/migrate/20221018121613_add_user_and_post_ref_to_likes.rb

```diff
@@ -0,0 +1,6 @@
+class AddUserAndPostRefToLikes < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :likes, :user, null: false, foreign_key: true
```

Comment:
- [ ] Based on feedback above kindly reference the author instead of user

Reply:
@julie-ify  Thank you for reviewing my project, I will make the necessary corrections as suggested